setup.py

```diff
@@ -22,10 +22,11 @@
     'paste',
     'passlib',
     'pycrypto',
-    'pyramid',
+    'pyramid~=1.7.5',
```

Comment:
Pyramid 1.8+ and webtest 2.0.24+ appear to have dropped support for Python 2.6, so I pinned their versions.